scripts/security/cve-template.json

```diff
@@ -0,0 +1,111 @@
+{
+    "CVE_data_meta": {
+        "ASSIGNER": "vulnerabilities@zephyrproject.org",
+        "DATE_PUBLIC": "2020-05-01T00:00:00.000Z",
+        "ID": "CVE-2020-XXXXX",
+        "STATE": "PUBLIC",
+        "TITLE": ""
+    },
+    "affects": {
+        "vendor": {
+            "vendor_data": [
+                {
+                    "product": {
+                        "product_data": [
+                            {
+                                "product_name": "zephyr",
+                                "version": {
+                                    "version_data": [
+                                        {
+                                            "version_affected": ">=",
+                                            "version_value": "1.14.0"
+                                        }
+                                    ]
+                                }
+                            }
+                        ]
+                    },
+                    "vendor_name": "zephyrproject-rtos"
+                }
+            ]
+        }
+    },
+    "credit": [
+        {
+            "lang": "eng",
+            "value": "NCC Group for report"
+        }
+    ],
+    "data_format": "MITRE",
+    "data_type": "CVE",
+    "data_version": "4.0",
+    "description": {
+        "description_data": [
+            {
+                "lang": "eng",
+                "value": "XXBuffer Overflow vulnerability in USB DFU of zephyr allows a USB connected host to cause possible remote code execution.\nThis issue affects:\nzephyrproject-rtos zephyr\nversion 1.14.0 and later versions."
```

Comment:
Since it is a template, we shouldn't have info about an specific issue.

Reply:
True. It gets replaced entirely.